container/images/eap8-domain-openjdk21-openshift/run-domain.sh

```diff
@@ -23,12 +23,6 @@ function run_setup_shutdown_hook() {
 }
 run_setup_shutdown_hook
 
-# Execute extensions
```

Comment:
Why is that removed and added later?

Reply:
To allow users to overwrite our default configurations via their post configure hooks.